contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '../../interfaces/IDCAStrategies.sol';
+
+abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  function getStrategy(uint80 _strategyId) external view override returns (Strategy memory) {}
```

Comment:
Since you already created the interfaces, could you add this to all functions in all handlers?
```suggestion
  /// @inheritdoc IDCAStrategiesManagementHandler
  function getStrategy(uint80 _strategyId) external view override returns (Strategy memory) {}
```